llama-index-core/llama_index/core/utilities/gemini_utils.py

```diff
@@ -17,17 +18,21 @@
     MessageRole.TOOL: MessageRole.USER,
     MessageRole.FUNCTION: MessageRole.USER,
 }
-ROLES_FROM_GEMINI: Dict[MessageRole, MessageRole] = {
```

Comment:
I think this worked only brecause `MessageRole` derives from `str` but let's be explicit